code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -0,0 +1,44 @@
+/mob/living/carbon/human/verb/chimera_revive() //Scree's race ability.in exchange for: No cloning.
+	set name = "Chimera Regenerate"
+	set category = "Abilities"
+
+	if(world.time < last_special)
```

Comment:
Or if regenerating? Wouldn't they be able to start two regens after 50 seconds? Maybe you should make this a clickcooldown thing. Look at belly_vr relayresist.